terraform/modules/gost_api/outputs.tf

```
@@ -47,3 +47,11 @@ output "efs_data_volume_access_point_id" {
 output "send_emails_policy_json" {
   value = module.send_emails_policy.json
 }
+
+output "digest_email_kickoff_cron_security_group_id" {
+  value = module.digest_email_kickoff_cron_security_group.id
+}
+
+output "digest_email_kickoff_cron_ecs_task_role_name" {
+  value = module.digest_email_kickoff_cron.ecs_task_role_name
+}
```

Per item 2a in our [Slack discussion](https://usdigitalresponse.slack.com/archives/C0324KDQSCR/p1699579161642309?thread_ts=1699568729.822169&cid=C0324KDQSCR):
```suggestion
output "digest_email_kickoff_cron_ecs_task_role_name" {
  value = join("", aws_iam_role.task.*.name)
}
```

Or, per item 2b (which I'm leaning toward, personally):
```suggestion
```